cli/azd/pkg/pipeline/pipeline_manager.go

```diff
@@ -289,19 +288,23 @@ func (pm *PipelineManager) Configure(ctx context.Context) (result *PipelineConfi
 	displayMsg = fmt.Sprintf("Configuring repository %s to use credentials for %s", repoSlug, spConfig.applicationName)
 	pm.console.ShowSpinner(ctx, displayMsg, input.Step)
 
+	subscriptionId := pm.env.GetSubscriptionId()
+	credentials := &azcli.AzureCredentials{
+		ClientId:       servicePrincipal.AppId,
+		TenantId:       *servicePrincipal.AppOwnerOrganizationId,
+		SubscriptionId: subscriptionId,
+	}
+
 	// Get the requested credential options from the CI provider
-	credentialOptions := pm.ciProvider.credentialOptions(
+	credentialOptions, error := pm.ciProvider.credentialOptions(
```

Comment:
```suggestion
	credentialOptions, err := pm.ciProvider.credentialOptions(
```
Can we call this `err` to be consistent with are default naming for errors?